package.json

```diff
@@ -63,12 +64,14 @@
     "xml-escape": "~1.0.0"
   },
   "devDependencies": {
+    "@ljharb/eslint-config": "^1.0.4",
```

Comment:
First I tried to keep the existing testing technique, faking things with `proxyquire`. But the `is-resolvable` module, added to determine whether a string indicates a file path or module ID, uses [`require.resolve`](https://nodejs.org/api/globals.html#globals_require_resolve) in its internal.

It seems that `proxyquire` doesn't support faking `require.resolve`. So, I added some actual scoped packages.